lib/components/player/stronzflix_player_controller.dart

```diff
@@ -48,15 +48,35 @@ abstract class StronzflixPlayerController {
     bool get isCompleted;
     StronzflixPlayerStream get stream;
 
+    bool get isEpisode => this._watchable is Episode;
```

Comment:
the video player should be agnostic of episodes and, more in general, any kind of subclass of watchable